test/map/map.test.cpp

```diff
@@ -321,6 +321,37 @@ TEST(Map, DefaultBoundOptions) {
     EXPECT_EQ(*bounds.bounds, LatLngBounds::unbounded());
 }
 
+TEST(Map, MapOptions) {
+    MapTest<> test;
+
+    test.map.setNorthOrientation(NorthOrientation::Rightwards);
+    test.map.setConstrainMode(ConstrainMode::None);
+    test.map.setViewportMode(ViewportMode::FlippedY);
+    Size size = { 512, 512 };
+    test.map.setSize(size);
+
+    auto options = test.map.getMapOptions();
+    EXPECT_EQ(options.mapMode(), MapMode::Static);
```

Comment:
This is not set above, should we remove this line since it is covered by `TEST(Map, DefaultMapOptions)`?

Reply:
I will update the test to set the map mode.